apis/v1/httproute_types.go

```diff
@@ -263,6 +263,11 @@ type HTTPRouteRule struct {
 	// invalid, 50 percent of traffic must receive a 500. Implementations may
 	// choose how that 50 percent is determined.
 	//
+	// When a HTTPBackendRef refers to a Service that has no ready endpoints,
+	// implementations SHOULD return a 503 for requests to that backend instead.
+	// If and implementation chooses to do this, all of the above rules for 500 responses
+	// MUST also apply.
```

Comment:
@dprotaso @youngnick Not sure I follow this - which of the following is the correct interpretation?

1. You can use a 503 in this case, but everything else should be precisely a 500.
2. If you use a 503 in this case, you should use a 503 anywhere else we've recommended using a 500.

I'm assuming it's the first one, but it's not completely obvious to me so could be worth clarifying the spec here.

Reply:
Yes, I intended the first one.

Reply:
Maybe change this to:

```suggestion
	// If an implementation chooses to do this, all of the above rules for 500 responses
	// MUST also apply for responses that return a 503.
```
(I just noticed the an/and typo, sorry!)